common/lib/knft-get-metadata.ts

```diff
@@ -1,14 +1,26 @@
 import type { NftType, NftMetadata } from "./ktypes";
 import { fetchJson } from "./kfetch";
-import { ipfsGetLink, ipfsGatewayUrl, getNetwork, getChecksumAddress, nftKey } from "./kconfig";
+import {
+  ipfsGetLink,
+  ipfsGatewayUrl,
+  swarmGetLink,
+  swarmGatewayUrl,
+  getNetwork,
+  getChecksumAddress,
+  nftKey
+} from "./kconfig";
+
+import { swarmGetContentType } from "./kbeejs";
 
 // Cache contentType(url)
 const contentTypes: Map<string, string> = new Map();
 
 const nftGetImageLink = (nft: NftType): string =>
```

Comment:
try this simplification : 

```
const nftGetImageLink = (nft: NftType): string =>
  ipfsGatewayUrl(nft?.ipfs) || swarmGatewayUrl(nft?.swarm) ||  nft?.image || "";
```

may require changes in ipfsGatewayUrl or swarmGatewayUrl... ?